releasenotes-builder/src/test/java/com/github/checkstyle/internal/AbstractPathTestSupport.java

```diff
@@ -42,6 +42,23 @@ public abstract class AbstractPathTestSupport {
     protected static final String MISC = Constants.MISCELLANEOUS_LABEL;
     protected static final String BREAKING = Constants.BREAKING_COMPATIBILITY_LABEL;
 
+    protected static final String MSG_EXECUTION_SUCCEEDED = System.lineSeparator()
+            + "Execution succeeded!" + System.lineSeparator();
+
+    protected static final String MSG_RELEASE_IS_MINOR =
+        "\nGeneration ends with 1 errors.\n\n[ERROR] Validation of release number failed. "
+            + "Release number is minor(10.0.0), but release notes do not contain 'new' or "
+            + "'breaking compatability' labels. Please correct release number by running "
+            + "https://github.com/checkstyle/checkstyle/actions/workflows/bump-version-and-"
+            + "update-milestone.yml\n";
+
+    protected static final String MSG_RELEASE_IS_PATCH =
+        "\nGeneration ends with 1 errors.\n\n[ERROR] Validation of release number failed. "
+            + "Release number is a patch(10.0.1), but release notes contain 'new' or "
```

Comment:
Both same here.

Reply:
Done.

Reply:
Not done.

Reply:
Done. Split into `getReleaseIsPatchMessage(String releaseVersion)`.